src/gatsby-node.js

```diff
@@ -1,20 +1,20 @@
-// const webpack = require(`webpack`)
+exports.resolvableExtensions = function() {
+  return [`.md`]
+}
 
-// Add Glamor support
-exports.modifyWebpackConfig = ({ config, stage }) => {
+exports.onCreateWebpackConfig = function({ actions }) {
   const mdFiles = /\.mdx?$/
-  config.loader(`mdx`, {
-    test: mdFiles,
-    loaders: [
-      'babel-loader?' + 'babelrc=false,' + 'presets[]=env,' + 'presets[]=react',
-      '@mdx-js/loader',
-    ],
+  actions.setWebpackConfig({
+    module: {
+      rules: [
+        {
+          test: mdFiles,
+          use: [
+            'babel-loader?' + 'babelrc=false,' + 'presets[]=env,' + 'presets[]=react',
```

Comment:
Did you run into issues if you didn't give `babel-loader` the query string params? @daneden

Reply:
Played around with this, what about the following?

```js
exports.onCreateWebpackConfig = ({ actions, loaders }) => {
  actions.setWebpackConfig({
    module: {
      rules: [
        {
          test: mdxTestRe,
          use: [loaders.js(), '@mdx-js/loader'],
        },
      ],
    },
  });
};
```

This uses `loader.js()` to give you what Gatsby already has.